src/Bridges/Constraint/bridges/fix_parametric_variables.jl

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    FixParametricVariablesBridge{T,S} <: AbstractBridge
+
+`FixParametricVariablesBridge` implements the following reformulation:
+
+  * ``f(x) \\in S`` into ``g(x) \\in S``, where ``g(x)`` is a
+    [`MOI.ScalarQuadraticFunction{T}`](@ref) and ``f(x)`` is a
+    [`MOI.ScalarAffineFunction{T}`](@ref) with all variables fixed using a
+    [`MOI.VariableIndex`](@ref)-in-[`MOI.EqualTo`](@ref) constraint replaced by
+    their corresponding constant.
+
+An error is thrown if, after fixing variables, ``f(x)`` is not an affine
+function,
+
+!!! warning
+    This transformation turns a quadratic function into an affine function by
+    substituting decision variables for their fixed values. This can cause the
+    dual solution of the substituted fixed variable to be incorrect. Therefore,
```

Comment:
Cant this work?

Reply:
I'm not sure. In the worst case, you could imagine the bridge substituting out every occurrence of the parameter, so the dual would be 0.